DiscordCode/FrontEnd/css/addFriend.css

```diff
@@ -0,0 +1,103 @@
+body {
```

Comment:
Looks good, may have to make adjusts to some colors later to provide overall consistency to the system.